start/main.py

```diff
@@ -19,15 +19,20 @@ def main():
                         prog='greptime-cloud-quick-start-python',
                         description='Quick start Python demo for greptime cloud')
 
-    parser.add_argument('-host', required=True, help='The host address of the GreptimeCloud service')
-    parser.add_argument('-db', '--database', required=True, help='The database of the GreptimeCloud service')
-    parser.add_argument('-u', '--username', required=True, help='The username of the database')
-    parser.add_argument('-p', '--password', required=True, help='The password of the database')
+    parser.add_argument('-host', default="localhost", help='The host address of the GreptimeDB')
```

Comment:
tip: use `-h` for consistency with other options

Reply:
`-h` is a keyword 🥲